Listenbrainz/UI/Screens/ContentView.swift

```diff
@@ -7,11 +7,13 @@ struct ContentView: View {
     let screenHeight = UIScreen.main.bounds.size.height
     @EnvironmentObject var homeViewModel: HomeViewModel
     @EnvironmentObject var feedViewModel: FeedViewModel
+    @StateObject var yimViewModel: YIMViewModel
 
     init() {
         UITabBar.appearance().layer.borderColor = UIColor.clear.cgColor
         UITabBar.appearance().clipsToBounds = true
-    }
+      _yimViewModel = StateObject(wrappedValue: YIMViewModel(repository: YIMRepositoryImpl()))
```

Comment:
Why are you adding the underscore here?

Reply:
@akshaaatt , It's a way of overriding the wrapped value inside a state object variable, however we can directly initialize repository in the State object, so removing it.

Reply:
Why would you ever try to override a vm? Also, if that is the case then why would you instantiate the StateObject again?

Reply:
:( my bad